Source/Core/DolphinWX/FifoPlayerDlg.cpp

```diff
@@ -80,197 +80,197 @@ void FifoPlayerDlg::CreateGUIControls()
 	m_Notebook = new wxNotebook(this, wxID_ANY);
 
 	{
-	m_PlayPage = new wxPanel(m_Notebook, wxID_ANY, wxDefaultPosition, wxDefaultSize, wxTAB_TRAVERSAL);
-	wxBoxSizer* sPlayPage;
-	sPlayPage = new wxBoxSizer(wxVERTICAL);
+		m_PlayPage = new wxPanel(m_Notebook, wxID_ANY, wxDefaultPosition, wxDefaultSize, wxTAB_TRAVERSAL);
```

Comment:
Indentation here isn't necessary, it's a block to group related objects together. Some threading locks also use this as well to limit the scope the lock is held.